.github/workflows/python_build.yaml

```diff
@@ -0,0 +1,85 @@
+name: Build Wheels
```

Comment:
Considering renaming this to "Create Offline Installation Archive" or something like that...open to suggesstions.